lib/main.dart

```diff
@@ -258,8 +260,9 @@ class MyHomePage extends StatelessWidget {
                 final taskTitle = titleController.text.trim();
                 final taskDescription = descriptionController.text.trim();
                 final taskDueDate = dueDateController.text.trim();
-
+                //Majd
                 if (taskTitle.isNotEmpty) {
+                  //Majd
```

Comment:
Consider creating a constructor in 'Task' class that accepts title, description, and due date as separate parameters for better clarity and data management.